src/Microsoft.NET.Sdk.Razor/build/netstandard2.0/Sdk.Razor.CurrentVersion.targets

```diff
@@ -247,7 +247,7 @@ Copyright (c) .NET Foundation. All rights reserved.
   <Target 
     Name="RazorGetAssemblyAttributes"
     AfterTargets="GetAssemblyAttributes"
-    Condition="'$(GenerateRazorAssemblyInfo)'=='true' and '$(RazorDefaultConfiguration)'!=''"
+    Condition="'$(GenerateRazorHostAssemblyInfo)'=='true' and '$(RazorDefaultConfiguration)'!=''"
```

Comment:
Shouldn't this be `GenerateRazorHostingAssemblyInfo`?

Reply:
Yup.

Reply:
Add a test?